### PR TITLE
Update schema repository for paper review

### PR DESCRIPTION
**Pull request by emi80**

### CODE REVIEW

File: testing/README.md [hunk 1]
```diff
@@ -21,13 +21,4 @@ The compliance and testing suite generates an HTML report for every tested serve
 
 * caltech.edu server - A demo server is available at http://felcat.caltech.edu/rnaget/ serving the Pan-Cancer Analysis of Whole Genomes (PCAWG) dataset.
```
Comment (saupchurch) [inline testing/README.md:22]: Please remove the link to the felcat server.  There have been some hardware issues and the server is currently down with no ETA on recovery.

File: README.md [hunk 2]
```diff
@@ -44,6 +44,15 @@ A group or organization can use the RNAget API to make their data available to u
...
 
+#### Client examples
+
+The client implementations below are provided by their respective developers as examples of using the API. GA4GH does not have a client testing suite as each client is an individual effort. Links to clients are presented as an "as is" resource for developers.
+
+| Developer | Client |
+|-----------|--------|
+| [@saupchurch](https://github.com/saupchurch) | an iPython notebook example designed to demonstrate basic navigation and operations on the demo PCAWG dataset.  It is available [here](https://github.com/saupchurch/bioinformatics-tools/blob/main/GA4GH-rnaget-API-examples.ipynb). An interactive version is accessible via [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/saupchurch/bioinformatics-tools/main?labpath=GA4GH-rnaget-API-examples.ipynb)|
```
Comment (saupchurch) [inline README.md:53]: This example no longer uses the PCAWG dataset.  It is a demonstrating basic operations using the GTEx implementation of the RNAget API.

File: testing/README.md (hunk 1 above)
Comment (emi80) [reply testing/README.md]: Updated in b119306

File: README.md (hunk 2 above)
Comment (emi80) [reply README.md]: Updated in c984981

**Review by saupchurch — APPROVED** [review]
Looks good to me.

**Review by emiliorighi — APPROVED** [review]

**Pull request merged**